.github/workflows/typescript-publish.yml

```
@@ -0,0 +1,39 @@
+    - name: Install
+      working-directory: typescript
+      run: |
+        pnpm install
+    - name: Run tests
+      working-directory: typescript
+      run: |
+        cd typescript
+        node --import=tsx --experimental-test-snapshots --test-concurrency=1 --test tests/*.test.ts
+    - name: Create Release Pull Request or Publish to npm
+      id: changesets
+      working-directory: typescript
```

actionlint (1.7.4)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)